JamGame/Assets/Scripts/TileBuilder/TileBuilderImpl.cs

```diff
@@ -440,5 +440,30 @@ public RoomCountByUid GetData()
 
             return new RoomCountByUid() { CountByUid = count };
         }
+
+        public Vector3 FitPointInside(Vector3 point)
```

Comment:
I'd better implement it as `GetBoundingBox` method as:
- Current impl is not performant enough to use it each frame when camera is moving (when bounding box can be easily cached)
- I believe that current impl will be not very intuitive when building have non-rectangular shape
- It'll be more generic